Configuration/FlexForms/flexform_publist_pi1.xml

```diff
@@ -300,15 +300,23 @@
          </numIndex>
          <numIndex index="14">
           <numIndex index="0">LLL:EXT:uma_publist/Resources/Private/Language/locallang_be.xlf:pi1_flexform.type.workshop_item</numIndex>
-          <numIndex index="1">adv:workshop</numIndex>
+          <numIndex index="1">workshop_item</numIndex>
```

Comment:
This fixes a problem with this option list by making sure that the names match to the ones defined everywhere else. Was there any reason for this prefix `adv` and different names used there?

Reply:
I think I added "adv:" just to make clear, that this is an extra attribute, which is not supported by madoc.

Reply:
Okay, but the prefix `adv:` is only used here. Therfore, these options didn't have an effect for me. It only worked when addin `ALL` as well. For example, it was not possible for me to reorder the publication types including workshop items. @sebeka Does that work for you?